packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -411,7 +411,17 @@ describe("DefaultVisualizers unit tests", () => {
 		const expected = {
 			value: "0",
 			nodeKind: "FluidValueNode",
-			tooltipContents: "com.fluidframework.leaf.number",
+			tooltipContents: {
```

Comment:
The number of changes you had to make here is an example of why I think it's really important we trim these tests down to much simpler cases. We should do that ASAP after this PR goes in.

Reply:
I will make a follow up task and work on this.